CONTRIBUTING.md

```diff
@@ -3,10 +3,10 @@ Thank you for your interest in contributing! Because we're a small team, we have
 
 ## Issues
 1. Issues are the best place to propose a new feature. Keep in mind that htmx is a small library, so there are lots of great ideas that don't fit in the core; it's always best to check in about an idea before doing a bunch of work on it.
-1. When proposing a new features, we will often suggest that you implement it as an [extension](https://htmx.org/extensions), so try that first. Even if we don't end up supporting it officially, you can publish it yourself and we can link to it.
+1. When proposing a new features, we will often suggest that you implement it as an [extension](https://htmx.org/docs/#extensions), so try that first. Even if we don't end up supporting it officially, you can publish it yourself and we can link to it.
```

Comment:
Think we forgot to update along with htmx 2 that moved extensions out of the main repo.
We probably want to directly point at [the extensions repo](https://github.com/bigskysoftware/htmx-extensions)  though.

Reply:
I can change that.